lib/friendly_shipping/services/ups_json/parse_rate_modifier_hash.rb

```diff
@@ -6,7 +6,7 @@ class UpsJson
       class ParseRateModifierHash
         # @param [Hash] hash the RateModifier hash from the source JSON
         # @param [String] currency_code The currency code for this modifier's amount (i.e. 'USD')
-        # @return [Hash]
+        # @return [Array] array The label and the amount of the rate modifier
```

Comment:
I don't think the lowercase "array" is necessary. The argument name is only necessary for arguments being passed into the method, not the return value.